lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -37,7 +37,9 @@ class DOM {
       element.className = className;
     }
     Object.keys(attrs).forEach(key => {
-      element.setAttribute(key, attrs[key]);
+      if (attrs[key] !== undefined) {
```

Comment:
I guess. You should document that behavior in that case so no one is surprised by it